webrender/src/resource_cache.rs

```diff
@@ -27,11 +26,24 @@ use webrender_traits::{GlyphDimensions, PipelineId, WebGLContextId};
 
 thread_local!(pub static FONT_CONTEXT: RefCell<FontContext> = RefCell::new(FontContext::new()));
 
+pub struct CacheItem {
+    pub texture_id: TextureId,
+    pub uv0: Point2D<f32>,
```

Comment:
would be great to specify that this is in texels, not actual normalized UVs, either by better naming, typing, or just comments